Makefile

```diff
@@ -41,6 +41,7 @@ bootstrap: toolset
 	mkdir -p $(BOOTSTRAP_PATH) ; \
 	cp Binaries/$(BUILD_CONFIGURATION)/csccore/* $(BOOTSTRAP_PATH) ; \
 	cp Binaries/$(BUILD_CONFIGURATION)/vbccore/* $(BOOTSTRAP_PATH) ; \
+	./build/scripts/crossgen.sh $(BOOTSTRAP_PATH) ; \
```

Comment:
On line 47 we invoke the `tests.sh` script using simply `/build/scripts` but here we use `./build/scripts` (the different is the leading `./`). 

Which is the correct way to do this?